shakenfist/daemons/main.py

```diff
@@ -53,7 +53,7 @@ def restore_instances():
                 LOG.info('%s Restoring instance' % i)
                 i.create()
             except Exception as e:
-                LOG.error('%s Failed to restore instance: %s' % (i, e))
+                LOG.error('Failed to restore instance: %s' % (e))
```

Comment:
So, we should be consistent about using instance at the front as a string, that's because instance has a __str__() method which returns "instance(uuid)" in a nice form. The pattern you're removing here is in the code base in many places.

Reply:
That said, maybe what we actually need is a logging helper to assist us with outputting high quality consistent messages so admins don't hate us?

Reply:
So the variable "i" cannot be reported since the exception can occur before initialisation.
The bug is actually that "instance" (not "i") should be in the log statement.
Yes?

Reply:
Oh I see. Fair.